source/Projectile.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+var Reference = require("Reference.js")
+
+export default class Projectile extends Pixi.Sprite {
+    constructor(x, y) {
+        var projectileTex =
+            Pixi.Texture.fromImage(require("./images/projectile.png"))
+        super(projectileTex)
+        this.x = x
+        this.y = y
+        this.speed = 80
+        console.log("Creating projectile")
```

Comment:
If nothing else, these `console.log`s were super useful in the playable build, so we can test that they're being destroyed. :)